mtp_cashbook/apps/cashbook/views.py

```diff
@@ -247,3 +254,119 @@ def get_context_data(self, **kwargs):
 
     def form_valid(self, form):
         return self.render_to_response(self.get_context_data(form=form))
+
+
+@method_decorator(login_required, name='dispatch')
+@method_decorator(expected_nomis_availability(True), name='dispatch')
+class ChangeNotificationView(TemplateView):
+    template_name = 'cashbook/change_notification.html'
+
+
+@method_decorator(login_required, name='dispatch')
+@method_decorator(expected_nomis_availability(True), name='dispatch')
+class NewCreditsView(FormView):
+    title = _('New credits')
+    form_class = ProcessNewCreditsForm
+    template_name = 'cashbook/new_credits.html'
+    success_url = reverse_lazy('new-credits')
+
+    def get_form_kwargs(self):
+        form_kwargs = super().get_form_kwargs()
+        form_kwargs['request'] = self.request
+        if 'ordering' in self.request.GET:
+            form_kwargs['ordering'] = self.request.GET['ordering']
```

Comment:
should this value have some validation? does the api silently ignore invalid ordering?